app.js

```diff
@@ -105,6 +105,9 @@ app.use(function(req, res, next) {
       !req.path.match(/^\/auth/) &&
       !req.path.match(/\./)) {
     req.session.returnTo = req.path;
+  } else if (req.user &&
+      req.path == '/account') {
+      req.session.returnTo = '/account';
```

Comment:
would you rather , I fold the condition in the existing if, instead of the else if?
aka:

```
  if ((!req.user &&
      req.path !== '/login' &&
      req.path !== '/signup' &&
      !req.path.match(/^\/auth/) &&
      !req.path.match(/\./)) 
      || (req.user &&
      req.path == '/account')
```